### DISCUSSION TITLE
数论——筛法求素数

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#229] [Announcements]
# 数论——筛法求素数

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2015/03/18/number-theory-sieve-method-to-find-primes/

**Comment by ktprime** [comment]
我的实现比你的开100-1000倍，如果n比较大。 性能与代码长度基本正正比
https://github.com/ktprime/ktprime/blob/master/PrimeNumber.cpp

**Reply by Xuanwo** [reply]
朋友，你成功地把我带回到 **6 年**前还在写 C 的快乐时光 :1st_place_medal:
